chart/values.yaml

```
@@ -1308,6 +1308,7 @@ dags:
     extraVolumeMounts: []
     env: []
     resources: {}
+    sshKeyCustomFile:
```

```suggestion
   # Path for the ssh key. Can be used with `sshKeySecret` to change the mount path or
   # used alone to have gitSync use another file (possibly via sidecar)
    sshKeyFile:
```

Not sure I like the name `sshKeyCustomFile`, I think `sshKeyFile` is better 🤷‍♂️.

---

I thought that too but not assigning it will confuse someone that it has the default value  of /etc/git-secret/ssh

---

i agree with @jedcunningham 

and maybe what  we can do here is, `sshKeyFile` can have an explicit default of `/etc/git-secret/ssh`

then we could reference this config instead of hardcoding e.g. in `_helpers.py` when setting `GIT_SSH_KEY_FILE`